package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/eth-ledger-bridge-keyring",
-  "version": "0.15.0",
+  "version": "0.16.0",
```

Comment:
Thoughts on making this v1 instead? @gantunesr and I are in agreement that it seems appropriate

Reply:
I've bumped the version, but I cannot change the branch name.

I've created a new PR with the correct branch name and all the changes discussed: https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/191